packages/api/src/resolvers/article/index.ts

```diff
@@ -434,7 +434,7 @@ export const getArticleResolver: ResolverFn<
 
     if (
       page.state === ArticleSavingRequestStatus.Processing &&
-      new Date(page.createdAt).getTime() < new Date().getTime() - 1000 * 60
+      new Date(page.createdAt).getTime() < new Date().getTime() - 1000 * 30
```

Comment:
Only reduced the duration here

Reply:
I think this should be `page.savedAt` because that is set each time the URL is saved.